blueocean-commons/src/main/java/io/jenkins/blueocean/commons/ListsUtils.java

```diff
@@ -0,0 +1,16 @@
+package io.jenkins.blueocean.commons;
+
+import java.util.Collection;
+
+public class ListsUtils {
+    public ListsUtils() {
+        // no op
+    }
+
+    public static <T>T getFirst( Collection<T> collection, T defaultValue){
+        if(collection==null){
+            return defaultValue;
+        }
+        return collection.isEmpty()?defaultValue:collection.iterator().next();
```

Comment:
```suggestion
        return collection.isEmpty() ? defaultValue : collection.iterator().next();
```

Reply:
new class so there is no formatting change..